eng/liveBuilds.targets

```diff
@@ -229,15 +227,15 @@
       
       <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
                              Include="
-        $(CoreCLRNativeArtifactsPath)dotnet.js;
-        $(CoreCLRNativeArtifactsPath)dotnet.js.map;
-        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js;
-        $(CoreCLRNativeArtifactsPath)dotnet.runtime.js.map;
-        $(CoreCLRNativeArtifactsPath)dotnet.d.ts;
-        $(CoreCLRNativeArtifactsPath)package.json;"
+        $(CoreCLRAotSdkDir)dotnet.js;
+        $(CoreCLRAotSdkDir)dotnet.js.map;
+        $(CoreCLRAotSdkDir)dotnet.runtime.js;
+        $(CoreCLRAotSdkDir)dotnet.runtime.js.map;
+        $(CoreCLRAotSdkDir)dotnet.d.ts;
+        $(CoreCLRAotSdkDir)package.json;"
         IsNative="true" />
       <LibrariesRuntimeFiles Condition="'$(TargetOS)' == 'browser' and '$(RuntimeFlavor)' == 'CoreCLR'"
-                             Include="$(CoreCLRNativeArtifactsPath)src\es6\*.js"
+                             Include="$(CoreCLRAotSdkDir)src\es6\*.js"
                              NativeSubDirectory="src\es6"
                              IsNative="true" />
```

Comment:
This should not be needed anymore. Also, `Microsoft.DotNet.ILCompiler.LLVM.pkgproj` needs to be fixed to not copy these files into `framework/`.